.ci/scripts/common.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Common functions for CI scripts
+#
+
+#
+# Check test script usage
+# Parse commandline arguments with first argument being the install directory.
+#
+check_install_dir() {
+    INSTALL_DIR=$1
+    if [ -z "$INSTALL_DIR" ]; then
+        echo "Usage: $0 <install_dir>"
+        exit 1
+    fi
+}
+
+#
+# Set environment variables for the build
+#
+set_env() {
```

Comment:
pls revert the unification of env vars, because:
1. this PR is indented for fixing TCP ports
2. the common varsity are not exactly as they were before the change (i.e cuda dirs) and we don't want to block review of the main fix because of other unrelated changes
3. we may want to remove these scripts altogether